backend/tests/test_db.py

```diff
@@ -29,18 +29,18 @@ def test_add_new_participant_to_db(_db, new_participant):
     assert len(db.session.query(Participant).all()) == 0
 
     new_participant = Participant(
-        name=new_participant['name'],
-        lastname=new_participant['lastname'],
+        first_name=new_participant['first_name'],
```

Comment:
We could do that as `new_participant = Participant(**new_participant)`

Reply:
changed :)